### DISCUSSION TITLE
Regarding AWS role arn to use instead of aws access key and secret key for installation

### DISCUSSION THREAD

**Discussion by andreacardoz22** [vmware-tanzu/velero#6282] [General discussions]
Hi Team,

Can you please help me with the documentation and latest on if role arn is supported instead of AWS access key and secret key for velero server installation.

Thanks

**Comment by mdnahian** [comment]
Hi,

There is no official documentation yet on how to setup Velero with IAM role. If you're using an EKS cluster there is a procedure posted [here](https://cloudcasa.io/blog/how-to-setup-velero-backups-on-eks-using-iam-roles-for-service-accounts-irsa/) to setup Velero using IRSA (IAM roles for service accounts) that might help.

Thanks,
Md ([cloudcasa.io](https://cloudcasa.io))
